packages/curriculum-compiler-json/index.js

```diff
@@ -63,6 +63,31 @@ function compileInsight(ast) {
   return json;
 }
 
+function compileQuestion(ast) {
+  if (!ast || !ast.children) {
+    throw new Error('Missing or invalid AST');
+  }
+
+  // Don't mutate parameter
+  const questionAst = Object.assign({}, ast);
+
+  const [heading] = questionAst.children.splice(
```

Comment:
why not just use `Array.prototype.find`? Why remove the heading?

```js
const heading = questionAst.children.find(node.type === 'heading')
```

Reply:
@nem035 Removing the `heading` node allows the rawText be the same as it would be if this question were a section of an insight... If you look at the commit messages I kind of explain how the iterations on this change.

Can add a comment if needed.